### ISSUE TITLE
using 

### ISSUE THREAD

**Issue by martinResearch** [akesterson/dpath-python#94]
Similarly to [python-benedict](https://github.com/fabiocaccamo/python-benedict) it would  would be nice to have a dictionary subclass that makes access to elements less verbose using for example this syntax

d = dpath_dict()
d['profile/firstname'] = 'Martin' # set values by keypath

**Comment by calebcase** [comment]
That would be neat!

**Comment by akesterson** [comment]
Blocked by #136
